### PR TITLE
Zoom/Hubspot Tests with MSW mocking of requests initiated from Next.js server

### PR DESCRIPTION
**Pull request by hariombalhara**
Fixes 
https://github.com/calcom/cal.com/issues/3177 
https://github.com/calcom/cal.com/issues/2255
## Tests that verify
- Zoom 
	- Installation/Uninstallation. 
	- Add zoom to an event-type and ensures that Zoom meeting is created
	- 2 Emails are sent with zoom links
- Hubspot
	- 	Installation/Uninstallation

### Approach
Tests with MSW require that playwright workers launch their own Next.js instance, so that within the tests we can mock requests based on the testcase

### Known Limitations -
- More than 1 workers don't currently work because our codebase doesn't support building once and then running with different ports. We need to use the port of the app itself when it was launched.
- Due to the above limitation I am running these tests in separate workflow.yml so that they don't slow down other tests.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -77,6 +77,14 @@ NEXT_PUBLIC_HELPSCOUT_KEY=
...
 SEND_FEEDBACK_EMAIL=
 
+# ZOOM
+ZOOM_CLIENT_ID="ZOOM_CLIENT_ID"
+ZOOM_CLIENT_SECRET="ZOOM_CLIENT_SECRET"
+
+# HUBSPOT
+HUBSPOT_CLIENT_ID="HUBSPOT_CLIENT_ID"
+HUBSPOT_CLIENT_SECRET="HUBSPOT_CLIENT_SECRET"
+
```
Comment (zomars) [inline .env.example:87]: AFAIK these should be on `.env.appStore`

**Review by zomars — COMMENTED** [review]
Code looking Gucci. Testing locally

**Comment by zomars** [comment]
Solved conflitcts and updated this PR @hariombalhara do you think this is mergable?

**Comment by hariombalhara** [comment]
@zomars yeah. It had forgotten to remove "Self review Pending" from description

**Comment by PeerRich** [comment]
are these only tests? should be safe to merge then. just noticed some merge conflicts @hariombalhara

**Comment by hariombalhara** [comment]
Yeah these are just tests. Will resolve the conflicts.

**Comment by hariombalhara** [comment]
Conflicts resolved.

**Comment by hariombalhara** [comment]
@zomars @zlwaterfield Please review this. Should be quick one as it affects only tests and they are passing.

**Review by zomars — APPROVED** [review]

**Pull request merged**